hydro_lang/src/sim/tests/snapshots/hydro_lang__sim__tests__trace_for_fuzzed_batching.snap

```diff
@@ -5,7 +5,12 @@ expression: log_str
 Running Tick
 * --> src/sim/tests/mod.rs:109:10
 *  |        .batch(&tick, nondet!(/** test */))
-*  |         ^ releasing items: [456, 456, 456, 456, 456, 456, 456, 456, ..] (1000 total)
+*  |         ^ releasing items: [456, 456, 456, 456, 456, 456, 456, 456, ..] (972 total)
```

Comment:
will need to revert once you undo the `.bin` change

Reply:
fixed